Scripts/ClearBinAndObjDirs.bat

```diff
@@ -0,0 +1,8 @@
+@echo off
+
+cd ..
```

Comment:
When a user run this script as an admin (they might), the working directory is changed to `C:\Windows\System32`. I suggest always adding `cd /d "%~dp0"` right after `@echo off` for every bat/cmd script.

Reply:
Thank you for advice. Check 1a7b96276dbaa66f47229a9965a59243afcab14d.